client/components/stylus_lib/index.styl

```diff
@@ -28,6 +28,7 @@ feature-body-w-margin = 720px
 feature-blockquote-width = 940px
 
 news-body-width = 780px
+news-body-width-w-margin = 820px
```

Comment:
Ohh now I get it -- "with margin" right?

Reply:
yeah, is to accommodate the margins for our hover drag/remove controls